java/src/main/java/gov/doc/ntia/sigmf/ext/global/sensor/Sensor.java

```diff
@@ -38,12 +39,20 @@ public class Sensor implements Serializable {
   @JsonProperty(value = "mobile", required = false)
   protected Boolean mobile;
 
+  @JsonProperty(value = "geolocation", required = false)
```

Comment:
This should not be added. The `core:geolocation` global field should be used instead

Reply:
^ I did this in ba17c4e988d6de7ec3541c6025c8c34b4595c4da